Build error with Nojde.js v10.21.0

When building `develop` branch with Node.js  v10.21.0 on Ubuntu20.04, it fails. See below about the log:
(top SHA = ea9ed34f3e85e9c66fc1a48dde95465cf596fa59)

```bash
../src/rcl_bindings.cpp: In function ‘std::unique_ptr<rmw_qos_profile_t> rclnodejs::GetQosProfileFromObject(v8::Local<v8::Object>)’:
../src/rcl_bindings.cpp:1225:39: error: no matching function for call to ‘v8::Value::BooleanValue(v8::Isolate*)’
 1225 |           currentContent->GetIsolate());
```

--- a/src/rcl_bindings.cpp
+++ b/src/rcl_bindings.cpp
@@ -1219,8 +1219,7 @@ std::unique_ptr<rmw_qos_profile_t> GetQosProfileFromObject(
   qos_profile->durability = static_cast<rmw_qos_durability_policy_t>(
       Nan::To<uint32_t>(durability).FromJust());
   qos_profile->avoid_ros_namespace_conventions =
-      avoid_ros_namespace_conventions->BooleanValue(
-          currentContent->GetIsolate());
+      Nan::To<bool>(avoid_ros_namespace_conventions).FromJust();
 
   return qos_profile;
 }

